schemas/iso19139/src/main/plugin/iso19139/formatter/jsonld/iso19139-to-jsonld.xsl

```diff
@@ -331,14 +309,66 @@
     <xsl:if test="count(gmd:identificationInfo/*/gmd:resourceConstraints/gmd:MD_LegalConstraints/gmd:otherConstraints[normalize-space(.) != '']) > 0"> 
       ,"license":  [<xsl:for-each 
         select="gmd:identificationInfo/*/gmd:resourceConstraints/gmd:MD_LegalConstraints/gmd:otherConstraints">
-          <xsl:apply-templates mode="toJsonLDLocalized" select="."/>
+        <xsl:choose>
+          <xsl:when test="starts-with(normalize-space(string-join(gco:CharacterString/text(),'')),'http') or starts-with(normalize-space(string-join(gco:CharacterString/text(),'')),'//')">
+            "<xsl:value-of select="normalize-space(string-join(gco:CharacterString/text(),''))"/>"
+          </xsl:when>
+          <xsl:when test="starts-with(string-join(gmx:Anchor/@xlink:href,''),'http') or starts-with(./@xlink:href,'//')">
+            "<xsl:value-of select="string-join(gmx:Anchor/@xlink:href,'')"/>"
+          </xsl:when>
+          <xsl:otherwise>
+            {
+              "@type": "CreativeWork",
```

Comment:
What is the reason to set the `@type` to the value `CreativeWork`?

Reply:
schema.org requires a url of creativework as value for license